Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -64,7 +68,22 @@ def run
         Homebrew.with_no_api_env do
           eval_all = args.eval_all? || Homebrew::EnvConfig.eval_all?
 
-          formulae_and_casks = if args.tap
+          formulae_and_casks = if args.auto?
+            tap_arg = args.tap
+            raise UsageError, "`--tap=` must be passed with `--auto`." if tap_arg.blank?
+            raise UsageError, "`--formula` or `--cask` must be passed with `--auto`." if !args.formula? && !args.cask?
+
+            tap = Tap.fetch(tap_arg)
+            autobump_list = tap.path/".github/autobump.txt"
```

Comment:
This is defined in `HOMEBREW_TAP_AUTOBUMP_FILE`. Should probably use this variable in `bump-*-pr.rb` too where it's hardcoded too.